epub34/authoring/index.html

```diff
@@ -498,14 +513,31 @@ <h3>Terminology</h3>
 					</dd>
 
 					<dt>
-						<dfn class="export" data-lt="xhtml content document|xhtml content documents">HTML content
-							document</dfn>
+						<dfn class="export">HTML content document</dfn>
```

Comment:
```suggestion
						<dfn class="export"  data-lt="xhtml content document|xhtml content documents">HTML content document</dfn>
```

Shouldn't we export the `data-lt` for the new text as well? We may want to add, if it becomes final, 'html content documents' as well

Reply:
I added a new dfn to export the term xhtml content document. We can't have that dfn exported and have a data-lt on this term to export the same term.

But the choice is whether to land all links at the primary term or to have documents still linking to xhtml content document land at the "formerly" line. I don't have a preference on that.

Reply:
Ah. Are we sure that the 'dfn' would cover the plural form as well? Isn't it why we used the data-lt attribute?

(I am fine leaving the xhtml ones on the 'formerly' line, b.t.w.)

Reply:
> Ah. Are we sure that the 'dfn' would cover the plural form as well?

Yes, respec [handles plurals automatically](https://respec.org/docs/#automatic-pluralization) so long as the term is in dfn tags. That's why we had to supply both forms of xhtml content doc to layer it onto the html content doc definition.

Reply:
Here's the tag respec creates for xhtml content document for example:

```
<dfn class="export" data-export="" data-plurals="xhtml content documents" id="dfn-xhtml-content-document" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">XHTML content document</dfn>
```

I don't know how it does the matching on data-plurals, but none of our terms has plurals in data-lt attributes and they've always linked up fine in plural form.